src/main/java/com/example/demo/Controller/StudentController.java

```diff
@@ -19,9 +18,27 @@ public class StudentController {
     private StudentService studentService;
 
       @GetMapping
+      @Tag(name = "List" )
     public List<Student> getAllStudents() {
 
           return studentService.getAllStudents();
     }
+    @GetMapping("/{id}")
+    @Tag (name = "List")
+    public Student getStudentById(@PathVariable Long id){
```

Comment:
Remember that discussed about that REST endpoints return multiple HTTP codes(and maybe Response Headers)?

If we write the method like this:
```
 public Student getStudentById(@PathVariable Long id){
 ```
 
 Then we will always return **200 as a HTTP response code** and with the default headers, even if we couldn't find the student by id.
 
 The correct behavior is to return 404 as a HTTP response code(NOT FOUND) if this student doesn't exist by id.
 
 https://developer.mozilla.org/en-US/docs/Web/HTTP/Status
 
The way to do this is to use the **ResponseEntity** wrapper class and wrap your response in it.
```
public Student getStudentById(@PathVariable Long id){
```

becomes

```

public ResponseEntity<Student> getStudentById(@PathVariable Long id){
  ResponseEntity.ok(body) .....
```